lm75bd/lm75bd.c

```diff
@@ -27,7 +27,23 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
+  uint8_t buf[2];
+
+  uint8_t addr = 0x0U;
+  i2cSendTo(devAddr, &addr, 1);
+
+  i2cReceiveFrom(devAddr, buf, sizeof(buf));
```

Comment:
Each function should be wrapped inside the `RETURN_IF_ERROR()` macro from the logging.h